oauth2_http/java/com/google/auth/oauth2/OAuth2Utils.java

```diff
@@ -70,21 +70,27 @@
 import java.util.Set;
 
 /** Internal utilities for the com.google.auth.oauth2 namespace. */
-class OAuth2Utils {
+public class OAuth2Utils {
```

Comment:
Since these were internal classes (package-private) and now public, thoughts on adding some strong-ish wording about how these are not intended for customers to use?

i.e. something
```
These classes are marked public but should treated effectively as internal classes only. These are not subject to any backwards compatibility requirements.
```